.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
we should include the same entries that are in the default golang gitignore for good measure https://github.com/github/gitignore/blob/main/Go.gitignore